modules/shortcodes/class.filter-embedded-html-objects.php

```diff
@@ -89,6 +89,10 @@ static public function filter( $html ) {
 		foreach ( $regexps as $element => $regexp ) {
 			self::$current_element = $element;
 
+			if ( ! is_string( $html )  || ! is_string( $element ) ) {
```

Comment:
I don't think it's necessary to include the second `is_string()` here -- $element is the key from the array, so there shouldn't be any issues with it.